commerce_coordinator/apps/titan/clients.py

```diff
@@ -223,6 +223,59 @@ def update_payment(self, payment_number, payment_state, response_code):
 
         return response['data']['attributes']
 
+    def create_payment(
+        self,
+        order_uuid,
+        response_code,
+        payment_method_name,
+        provider_response_body,
+        reference=None,
+        amount=None,
+        payment_date=None,
+        source=None,
+    ):
+        """
+        Request Titan to Create payment.
+
+        Args:
+            order_uuid(str): Order UUID related to this order.
+            response_code(str): Payment attempt response code (payment intent id) provided by stripe.
+            payment_method_name(str): The name of the payment method used for this payment. See enums for valid values.
+            provider_response_body(str): The response JSON dump from a request to the payment provider..
+            reference(str): Optional. Reference to be saved against the payment.
```

Comment:
[upstream spec question] I'm not sure what the difference will be between `reference` and `response_code`.

Reply:
Reference is an arbitrary string used internally (by our Finance team) for reconciliation as far as I know. Response code is the intent client key from Stripe

Reply:
So this field will be auto-generated by Titan's system? Commerce Coordinator do not need to send it to Titan in their create payment API call?